### PR TITLE
Settings and initial contents for Italian localization

### PR DESCRIPTION
**Pull request by sistella**
Together with https://github.com/cncf/glossary/pull/375 (Style-Guide), this is the PR to set the Italian website

### CODE REVIEW

**Comment by seokho-son** [comment]
@sistella ! Thank you for the contribution ! :)

We are going to check following tasks in this PR. (target branch is dev-it)

- [ ] 3-1. Adding a new language setting to site configuration https://github.com/cncf/glossary/blob/main/LOCALIZATION.md#3-1-adding-a-new-language-setting-to-site-configuration
- [ ] 3-2. Adding a file for site strings https://github.com/cncf/glossary/blob/main/LOCALIZATION.md#3-2-adding-a-file-for-site-strings
- [ ] 3-3. Adding a new localization directory https://github.com/cncf/glossary/blob/main/LOCALIZATION.md#3-3-adding-a-new-localization-directory
- [ ] 3-4. Minimum required content https://github.com/cncf/glossary/blob/main/LOCALIZATION.md#3-4-minimum-required-content
- [ ] 3-5. Check configuration works https://github.com/cncf/glossary/blob/main/LOCALIZATION.md#3-5-check-configuration-works-and-open-a-pr

File: config.toml [hunk 1]
```diff
@@ -58,6 +58,14 @@ description = "The CNCF Cloud Native Glossary Project is intended to be used as
 languageName ="English"
 # Weight used for sorting.
 weight = 1
+
+[languages.it]
+title = "Glossario Cloud Native"
+description = "Il progetto del Glossario CNCF Cloud Native è pensato per essere un riferimento per i termini comuni, usati nell'ambito delle applicazioni cloud native."
+languageName = "Italiano"
+# Weight used for sorting.
```
Comment (seokho-son) [inline config.toml:66]:
```suggestion
contentDir =  "content/it"
```
This is important parameter :) 

You can check the preview of the site (https://deploy-preview-404--cncfglossary.netlify.app/)
The preview does not show Italian contents.

**Review by seokho-son — CHANGES_REQUESTED** [review]

**Comment by seokho-son** [comment]
@sistella @annalisag-spark Could you add the style-guide/_index.md file in #375 to this PR? 
to fulfill [`3-4. Minimum required content`](https://github.com/cncf/glossary/blob/main/LOCALIZATION.md#3-4-minimum-required-content) in one PR.

- Home: content/en/_index.md -> content/<localization>/_index.md
- How to contribute: content/en/contribute/_index.md -> content/<localization>/contribute/_index.md
- Style guide: content/en/style-guide/_index.md -> content/<localization>/style-guide/_index.md

**Review by seokho-son — APPROVED** [review]

**Comment by sistella** [comment]
@seokho-son Style-guide is here!!

**Review by annalisag-spark — APPROVED** [review]
Looks good to me!

**Comment by seokho-son** [comment]
@sistella 
The file name and path for style doc seems odd.

Current file name and path: [`annalisag-spark:terms/style-guide`](https://github.com/cncf/glossary/pull/404/files#diff-f7bf3db3085a8bebdd76ef38a97fccd1f96a5ceb7b4b7568bbb6a05dc3833941)

it should be 
`content/it/style-guide/_index.md`

**Review by seokho-son — APPROVED** [review]
LGTM :)

- [x] 3-1. Adding a new language setting to site configuration https://github.com/cncf/glossary/blob/main/LOCALIZATION.md#3-1-adding-a-new-language-setting-to-site-configuration
- [x] 3-2. Adding a file for site strings https://github.com/cncf/glossary/blob/main/LOCALIZATION.md#3-2-adding-a-file-for-site-strings
- [x] 3-3. Adding a new localization directory https://github.com/cncf/glossary/blob/main/LOCALIZATION.md#3-3-adding-a-new-localization-directory
- [x] 3-4. Minimum required content https://github.com/cncf/glossary/blob/main/LOCALIZATION.md#3-4-minimum-required-content
- [x] 3-5. Check configuration works https://github.com/cncf/glossary/blob/main/LOCALIZATION.md#3-5-check-configuration-works-and-open-a-pr

**Comment by seokho-son** [comment]
PTAL @CathPag

File: content/it/_index.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+
+---
+title: "Glossario Cloud Native"
+---
+
+# Glossario Cloud Native
+
+Il Glossario Cloud Native è un progetto guidato dal Business Value Subcommittee (BVS) della CNCF con cui la community si propone di spiegare concetti del mondo cloud native con un linguaggio semplice e diretto che non presupponga conoscenze tecniche pregresse. La [versione in PDF può essere visualizzata e scaricata qui] (https://github.com/cncf/glossary/blob/main/cloudnative-glossary.pdf).
```
Comment (CathPag) [inline content/it/_index.md:8]: We removed the last sentence with the reference to the PDF as it's outdated.

File: content/it/_index.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+
+## Contribuire
+
+L'invito a suggerire modifiche, aggiunte e migliorie al Glossario Cloud Native è aperto a chiunque abbia voglia di contribuire. Il processo che adottiamo per lo sviluppo e l'arricchimento di questo dizionario condiviso è guidato dalla community e governato dalla CNCF. Il Glossario fornisce una piattaforma vendor-neutral per organizzare un vocabolario condiviso riguardante le tecnologie cloud native. 
+Chiunque voglia contribuire può aprire una issue o una pull request su GitHub. Leggi la sezione [Come Contribuire](/contribuire/) per maggiori informazioni e assicurati di seguire le indicazioni della [Style Guide](/style-guide/).
```
Comment (CathPag) [inline content/it/_index.md:14]: I think a space is missing here or did you want it to be in one paragraph. Adding the text in the next line will still show it as one paragraph. It works as one as well but wanted to point it out. Up to you really :)

File: content/it/_index.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+
+## Riconoscimenti
+
+Il progetto del Glossario Cloud Native è un'iniziativa del Marketing Committee della CNCF (nello specifico del Business Value Subcommittee) e include contributi da [Catherine Paganini](https://www.linkedin.com/in/catherinepaganini/en/), [Chris Aniszczyk](https://www.linkedin.com/in/caniszczyk/),
+[Daniel Jones](https://www.linkedin.com/in/danieljoneseb/?originalSubdomain=uk), [Jason Morgan](https://www.linkedin.com/in/jasonmorgan2/), [Katelin Ramer](https://www.linkedin.com/in/katelinramer/) e [Mike Foster](https://www.linkedin.com/in/mfosterche/?originalSubdomain=ca).
```
Comment (CathPag) [inline content/it/_index.md:20]: There has been an update here. Seokho and the list of all contributors is missing

> The Cloud Native Glossary was initiated by the CNCF Marketing Committee (Business Value Subcommittee) and includes contributions from [Catherine Paganini](https://www.linkedin.com/in/catherinepaganini/en/), [Chris Aniszczyk](https://www.linkedin.com/in/caniszczyk/), [Daniel Jones](https://www.linkedin.com/in/danieljoneseb/?originalSubdomain=uk), [Jason Morgan](https://www.linkedin.com/in/jasonmorgan2/), [Katelin Ramer](https://www.linkedin.com/in/katelinramer/), [Mike Foster](https://www.linkedin.com/in/mfosterche/?originalSubdomain=ca), [Seokho Son](https://www.linkedin.com/in/seokho-son/) and many more contributors. For a complete contributor list, please refer to [this GitHub page](https://github.com/cncf/glossary/graphs/contributors).

File: content/it/_index.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+
+## Riconoscimenti
+
+Il progetto del Glossario Cloud Native è un'iniziativa del Marketing Committee della CNCF (nello specifico del Business Value Subcommittee) e include contributi da [Catherine Paganini](https://www.linkedin.com/in/catherinepaganini/en/), [Chris Aniszczyk](https://www.linkedin.com/in/caniszczyk/),
+[Daniel Jones](https://www.linkedin.com/in/danieljoneseb/?originalSubdomain=uk), [Jason Morgan](https://www.linkedin.com/in/jasonmorgan2/), [Katelin Ramer](https://www.linkedin.com/in/katelinramer/) e [Mike Foster](https://www.linkedin.com/in/mfosterche/?originalSubdomain=ca). 
+La localizzazione del progetto in italiano è a cura di [Giulia Di Pietro](https://www.linkedin.com/in/giulia-dp/), [Meryem Fourdaous](https://www.linkedin.com/in/meryem-fourdaous-022346102/), [Annalisa Gennaro](https://www.linkedin.com/in/annalisagennaro/) [Francesco Sbaraglia](https://www.linkedin.com/in/fsbaraglia/) e [Simone Stella](https://www.linkedin.com/in/simostella/).
```
Comment (CathPag) [inline content/it/_index.md:21]: Same paragraph thing here. I feel that with a separate paragraph, your names will better highlighted vs buried at the end of the paragraph. Again up to you.

**Review by CathPag — CHANGES_REQUESTED** [review]
We need to remove one sentence and add one. I also added two suggestions, feel free to ignore them if you feel it works better the way it is. Other than that, looks great!!

File: content/it/_index.md (hunk 2 above)
Comment (sistella) [reply content/it/_index.md:8]: done!

File: content/it/_index.md (hunk 4 above)
Comment (sistella) [reply content/it/_index.md:20]: done

File: content/it/_index.md (hunk 5 above)
Comment (sistella) [reply content/it/_index.md:23]: Yeah, it's way better, thanks @CathPag

**Pull request merged**